x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -399,25 +404,27 @@ export const getMonitoringColumns = (columnsProps: GetColumnsProps): TableColumn
           }
         />
       ),
-      render: (value: Rule['last_gap'] | undefined) => (
+      render: (value: DurationMetric | undefined) => (
         <EuiText data-test-subj="gap" size="s">
-          {value ?? getEmptyTagValue()}
+          {value != null ? value.toFixed() : getEmptyTagValue()}
```

Comment:
I think it's worth to display, that gap is measured in seconds, to end users. Or even format seconds into more friendly format(minutes, hr, etc depends on a gap value)

With this change, user will see only number in a gap column, with no indication to what this number refers to.
Screenshot with gaps:
<img width="1259" alt="Screenshot 2022-01-17 at 14 04 03" src="https://user-images.githubusercontent.com/92328789/150010331-487dc2d7-dd82-44ef-8c4f-62291a936a3c.png">

Reply:
Great catch @vitaliidm 👍  I'll address this in a follow-up PR and I think we should format it in a human-friendly way (e.g. `12h 07m`)